File: problem_thirty_one.py
# multiplier value1 to value2
value1 = int(input("Enter the lower number:"))
value2 = int(input("Enter the upper number:"))
def Multiplier(input1,input2):
    if input1 < input2:
        count = 0
        count1 = 1
        while count <= input2:
            value = input1 * count1
            if value <= input2:
                print(value)
            count = value
            count1+=1
    else:
        print("invalid")        

Multiplier(value1,value2)

File: problem_thirty_one.js
// Multiplier inp1 to inp2
var inp1 = 2
var inp2 = 10
function Multiplier(inp1,inp2){
    if (inp1 < inp2){
       var count  = 0
        var range = 1
        while(count <= inp2){
            var value =  inp1*range
            if(value <= inp2){
                console.log(value)
            }
            count = value
            range+=1
        }
    }else{
        console.log("invalid")
    }
    }
  
Multiplier(inp1,inp2)
